redbox-core/redbox/models/settings.py

```diff
@@ -56,34 +47,13 @@ class ChatLLMBackend(BaseModel):
 class Settings(BaseSettings):
     """Settings for the redbox application."""
 
-    embedding_openai_api_key: str = "NotAKey"
```

Comment:
I imagine it's not strictly necessary, but should we remove these from the terraform code (variables and then where those variables are called)?

Reply:
good point but we are either using:
* Azure/OpenAI: which is not managed under terraform
* Bedrock: which is managed under terraform but at a team rather than a project level

Reply:
Does that mean even more reason to remove them? Why are they currently defined as variables in terraform?

Reply:
yes - good point - i hadnt spotted that - ill remove